src/build_infrastructure/android/README.md

```diff
@@ -0,0 +1,69 @@
+## Usage
+
+Start from the CDK root, where this `README.md` lives:
+```console
+cd ~/amplify-ci-support/src/build_infrastructure_resources/android
+```
+
+Ensure that you have the CDK installed, if you haven't yet.
+```console
+npm install -g aws-cdk
+```
+
+Ensure that you have [credentials in your environment sufficient to run
+the CDK](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_credentials).
+
+Install CDK dependencies from the root directory of the repo:
+```console
+# Best to do this in a virtualenv
+pip3 install -r requirements.txt
+```
+
+**Export credentials and region:**
+
+Export appropriate role credentials to your local shell. Specifically, you must the following environment variables set:
+1. `AWS_ACCESS_KEY_ID`
+2. `AWS_SECRET_ACCESS_KEY`
+3. `AWS_SESSION_TOKEN`
+
+Also, you must set the AWS region to us-east-1:
+
+```console
+export AWS_DEFAULT_REGION=us-east-1
+```
+
+**Bootstrap the CDK**
+If you have not yet, bootstrap the CDK:
+```console
+cdk bootstrap
+```
+
+### Deploy all stacks
+
+Note that these steps do each take a _while_.
+```console
+list=$(cdk list 2>/dev/null | xargs)
```

Comment:
For this project, are there multiple targets? Or can we just do one?

Reply:
Currently there is only one. However, I think we might end up creating a separate pipeline to handle releases.